src/appleseed.studio/mainwindow/project/projectexplorer.cpp

```diff
@@ -115,6 +115,8 @@ ProjectExplorer::ProjectExplorer(
     connect(
         &m_project_builder, SIGNAL(signal_frame_modified()),
         SIGNAL(signal_frame_modified()));
+
+    m_material_drop_handler.reset(new MaterialDropHandler(project, rendering_manager));
```

Comment:
In my idea, `MaterialDropHandler` was created by `RenderTab`, just like all other handlers. That would simplify things even more.

If that means you'll need to pass the `RenderingManager` to `RenderTab`, then fine.